api/migrations/schema/Version20230409164830.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20230409164830 extends AbstractMigration {
+    public function getDescription(): string {
+        return 'Create ActivityProgressLabels for camps from their prototype';
+    }
+
+    public function up(Schema $schema): void {
+        $this->addSql('CREATE EXTENSION pgcrypto');
+
+        // Test:
+        // J+S
```

Comment:
Das ist für das Template. Wollen wir dem GRGR-Lager und dem Harry-Potter Lager auch noch ein paar Default-Werte spendieren? Muss für mich nicht zwingend in eine Migration. Kann auch direkt in den dev-data Dump.